style.css

```diff
@@ -4,110 +4,96 @@ body {
 }
 
 .menu {
-    
+    display: block;
+    text-align: center;
     width: 1920px;
     height: 1080px;
     background: linear-gradient(0deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url(bg-images/1.jpg);
 }
 
-.bigsun {
-    position: absolute;
-    top: 365px;
-    left: 900px;
-    color: #FFFFFF;
+.block__menu {
+    padding-top: 347px;
 }
 
-h1 {
-    position: absolute;
-    width: 492px;
-    height: 46px;
-    top: 489px;
-    left: 714px;
+.title__one {
     font-family: 'Reem Kufi Ink';
     font-style: normal;
     font-weight: 400;
     font-size: 40px;
     line-height: 60px;
-    text-align: center;
     letter-spacing: 0.015em;
```

Comment:
If you're using the same styles in each text block then declare it in the `body` or another top-level container